src/docs/rfcs/0000_remove_object_concept.md

```diff
@@ -0,0 +1,118 @@
+- Proposal Name: `remove_object_concept`
+- Start Date: `2023-03-05`
+- RFC PR: [datafuselabs/opendal#0000](https://github.com/datafuselabs/opendal/pull/0000)
+- Tracking Issue: [datafuselabs/opendal#0000](https://github.com/datafuselabs/opendal/issues/0000)
+
+# Summary
+
+Remove the Object concept to make OpenDAL easier to understand.
+
+# Motivation
+
+OpenDAL introduces [Object Native API][super::super::0041_object_native_api] to resolve the problem of not easy to use:
+
+```diff
+- let reader = SeekableReader::new(op, path, stream_len);
++ let reader = op.object(&path).reader().await?;
+
+- op.stat(&path).run().await
++ op.object(&path).stat().await
+```
+
+However, times changing. Aftre `list` operation has also been moved to `Object` level, `Object` is more like a wrapper of `Opeator`. And the only meaning API of `Opeator` is `Opeator::object`.
```

Comment:
typo: `Opeator` -> `Operator`

Reply:
typo: `Aftre` -> `After`

suggestion: 
- ``After the `list` operation``
- the only **meaning** API -> the only **meaningful** API